tests/integration/targets/test_move_objects_between_storage_buckets/tasks/create_buckets.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Create GCP Storage bucket
+  google.cloud.gcp_storage_bucket:
+    name: "{{ item }}"
+    auth_kind: "{{ gcp_auth_kind }}"
+    service_account_file: "{{ gcp_cred_file }}"
+    project: "{{ gcp_project }}"
```

Comment:
Why don't you use the role for creds setting here?

Reply:
I can also do that, I was just using it for collection role, but it should work too